src/instana/tracer.py

```diff
@@ -118,7 +118,7 @@ def start_span(
         record_exception: bool = True,
         set_status_on_exception: bool = True,
     ) -> InstanaSpan:
-        parent_context = span_context
+        parent_context = get_current_span().get_span_context() if span_context is None else span_context
```

Comment:
I think it's cleaner and easier to read this way. What do you think @GSVarsha ?

```suggestion
        parent_context = span_context if span_context else get_current_span().get_span_context()
```

Reply:
Done!